### PR TITLE
Add spdlog version 1.10.0

### PR DESCRIPTION
**Pull request by Kernald**

### CODE REVIEW

**Comment by Kernald** [comment]
The failure on CI at the moment is caused by ftm-8 not existing - which is correct. I assumed bzlmod would pick the existing 8.1.1, as any 8.x would work for spdlog 1.10.0. Should I go ahead and set 8.1.1 specifically, or is there another kind of notation that I could use here?

**Comment by fmeum** [comment]
bzlmod doesn't interpret the version string according to semver conventions (see https://bazel.build/docs/bzlmod#version-format). As a result, all versions you specify must actually exist in the registry.

**Comment by Kernald** [comment]
Thanks! I just fixed the version then.

File: modules/spdlog/metadata.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "homepage": "https://github.com/gabime/spdlog",
+    "maintainers": [],
```
Comment (meteorcloudy) [inline modules/spdlog/metadata.json:3]: @Kernald Do you want to list yourself as a maintainer of this module?

**Review by meteorcloudy — APPROVED** [review]

File: modules/spdlog/metadata.json (hunk 1 above)
Comment (meteorcloudy) [reply modules/spdlog/metadata.json:3]: OK, we'll leave it empty for now until someone volunteers.

**Pull request merged**